css/style.css

```diff
@@ -30,7 +30,7 @@ header {
   margin: 0 auto;
   padding: 15px;
   background-color: #67cae9;
-  border-radius: 5px;
+  border-radius: 555px;
```

Comment:
Please turn down border-radius pixels to 5 px to stick the portrait into the whole design.